app/javascript/widget/components/ChatAttachment.vue

```diff
@@ -65,6 +71,10 @@ export default {
           this.$refs.upload.add(file);
         }
       });
+      // Clear quick replies options in case the user ignores the quick replies
```

Comment:
This can be a function named: 
	
```
	checkIfOptionSelected() {
	      if (this.quickRepliesOptions.length) {
        this.setQuickRepliesOptions([]);
      }
	}
```

Reply:
Agree, fixing it!